### ISSUE TITLE
为什么UNet的最后一层kernel_size=3

### ISSUE THREAD

**Issue by ql390962** [PaddlePaddle/PaddleSeg#2026]
self.encode = Encoder()
        self.decode = Decoder(align_corners, use_deconv=use_deconv)
        self.cls = self.conv = nn.Conv2D(
            in_channels=64,
            out_channels=num_classes,
            kernel_size=3,
            stride=1,
            padding=1)

        self.pretrained = pretrained
        self.init_weight()

    def forward(self, x):
        logit_list = []
        x, short_cuts = self.encode(x)
        x = self.decode(x, short_cuts)
        logit = self.cls(x)
        logit_list.append(logit)
        return logit_list
我改成1后会报错，torch写的也是等于1，为什么这里是3，好奇怪。

**Comment by wuyefeilin** [comment]
你可以根据自己的需求进行魔改哈
